documentation/README.md

```diff
@@ -0,0 +1,21 @@
+# Documentation for the Grants Equity Project
+
+This folder contains documentation for the Grants Equity Project. The following are some important, high-level 
```

Comment:
I think this sentence is unfinished. I'd add a suggestion but not sure your vision

Reply:
Thanks, I updated it.